### PR TITLE
Fix TypeError for null objects (#12)

### PR DESCRIPTION
**Pull request by mbelsky**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -14,8 +14,10 @@ const set = require('./');
 
 describe('set', function() {
   it('should return non-objects', function() {
-    const res = set('foo', 'a.b', 'c');
+    var res = set('foo', 'a.b', 'c');
```
Comment (davesag) [inline test.js]: better to use `let` than `var` to avoid hoisting.

Comment (mbelsky) [reply test.js]: As I see the file has `const` and `var` declarations only, so I've used `var` for consistency

Comment (davesag) [reply test.js]: fair enough

**Comment by mbelsky** [comment]
Hey @jonschlinkert 
Could you please review this small pr?

**Comment by jonschlinkert** [comment]
Thanks @mbelsky!

**Pull request merged**